source/jormungandr/tests/check_utils.py

```diff
@@ -517,6 +528,8 @@ def is_valid_graphical_isochrone(isochrone, tester, query):
         geojson = g['geojson']
         assert geojson
         is_valid_multipolygon_geojson(geojson)
+        get_valid_unsigned_int(g['max_duration'])
+        get_valid_unsigned_int(g['min_duration'])
```

Comment:
you should also add in `is_valid_graphical_isochrone` that `from` xor `to` is present.